src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -1,4 +1,38 @@
-const convertInclude = ({ href, url, ...rest }) => {
+import 'isomorphic-fetch';
+import webLogger from '#lib/logger.web';
+
+const logger = webLogger();
+
+// this defaults all VJ and IDT2 includes to the specified urls for now
+// should be removed once mozart routes have been created
+const includeUrls = {
+  include: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/vj.html',
+  idt2: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt2.html',
+};
+
+const encodeHTML = str =>
+  String(str)
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;');
```

Comment:
I'm a bit unsure where the encoding is necessary? I thought something in the express middleware would be clever enough to do this for you?

Looks like you may have missed single quotes (`'`) btw: https://stackoverflow.com/questions/40263803/native-javascript-or-es6-way-to-encode-and-decode-html-entities

Reply:
> Looks like you may have missed single quotes (') btw: https://stackoverflow.com/questions/40263803/native-javascript-or-es6-way-to-encode-and-decode-html-entities

Thanks for this.

> I'm a bit unsure where the encoding is necessary?

Passing the html text from the server distorts the hydration as the text contains un-escaped literals. It would render but the markup will also appear at the bottom of the page and any property attached on the `window` object on the server becomes `undefined`.